app/javascript/packs/brave_rewards_banner_display.jsx

```diff
@@ -237,8 +278,10 @@ class BraveRewardsPageForm extends React.Component {
     const body = new FormData();
 
     body.append('title', document.getElementsByClassName("sc-gZMcBi")[0].innerText);
-    body.append('description', document.getElementsByClassName("sc-gqjmRU")[0].innerText);
+    body.append('title', document.getElementsByClassName("sc-gZMcBi")[0].innerText);
+    body.append('description', this.state.description);
     body.append('donation_amounts', JSON.stringify(this.state.donationAmounts));
+    body.append('social_links', JSON.stringify(this.state.socialLinks));
```

Comment:
So `JSON.stringify` converts a hash into json string, and when it saves, it saves into the database, it saves a json string as opposed to a json. I'm not sure if we should avoid converting here or do a `JSON.parse` the controller

Reply:
I think it's common to stringify here, but I implemented your suggestion to parse in the controller and that really helped clean the code.